### DISCUSSION TITLE
The ability to submit alt text to other people's image

### DISCUSSION THREAD

**Discussion by ghost** [bluesky-social/atproto#1585] [Bluesky Lexicons]
**Is your feature request related to a problem? Please describe.**
There's not a lot of people putting alt text on their images.

**Describe the solution you'd like**
Letting people submit alt text to other people's images.  A button in the three dot menu to submit alt text could work.

**Comment by myConsciousness** [comment]
@Naverim 

The decision to add Alt Text would be the will of the owner of the image and not the will of other users. Also, I'm sure this will be abused.

**Comment by ghost** [comment]
There could be a system where the owner of the image can approve on the submitted alt text. The owner could report inappropriate user submitted alt-text and the dev suspend them from submitting alt text ever again.
